bin/install-site.sh

```diff
@@ -283,6 +283,22 @@ deb-src http://security.debian.org/ wheezy/updates main non-free
 deb http://http.debian.net/debian wheezy-backports main contrib non-free
 deb-src http://http.debian.net/debian wheezy-backports main contrib non-free
 EOF
+    elif [ x"$DISTRIBUTION" = x"debian" ] && [ x"$DISTVERSION" = x"jessie" ]
+    then
+        # Install the basic packages we require:
+        cat > /etc/apt/sources.list.d/mysociety-extra.list <<EOF
+    # Debian mirror to use, including contrib and non-free:
+    deb http://the.earth.li/debian/ jessie main contrib non-free
+    deb-src http://the.earth.li/debian/ jessie main contrib non-free
```

Comment:
No, no reason that I know of, or can remember. I might just have been using the sources from the base box I was using.